python/tests/integration/completions/test_chat_completions.py

```diff
@@ -406,6 +409,17 @@ def services() -> dict[str, tuple[ChatCompletionClientBase, type[PromptExecution
             marks=pytest.mark.skipif(not mistral_ai_setup, reason="Mistral AI Environment Variables not set"),
             id="mistral_ai_text_input",
         ),
+        pytest.param(
+            "ollama",
+            {},
+            [
+                ChatMessageContent(role=AuthorRole.USER, items=[TextContent(text="Hello")]),
+                ChatMessageContent(role=AuthorRole.USER, items=[TextContent(text="How are you today?")]),
+            ],
+            ["Hello", "well"],
+            marks=pytest.mark.skip(reason="Need local Ollama setup"),
```

Comment:
Is it worth it to add this test here? Do we plan on adding an Ollama set up as part of the GitHub actions to run the tests? Or is this for a dev to run the test locally?

Reply:
I am thinking we just do it for Linux since it's easier to install. But I am still investigating.

Reply:
Setup our integration test pipeline to install Ollama via docker.